e2e/pages/strategy.spec.ts

```diff
@@ -14,57 +17,46 @@ test.describe('Strategies', () => {
   });
 
   test('Create Limit Strategy ETH->DAI', async ({ page }) => {
-    test.setTimeout(120_000);
-    await waitFor(page, 'balance-DAI', 20_000);
+    test.setTimeout(180_000);
+    const config = {
```

Comment:
absolute total nitpick, but if you put the config outside of the test then the test name can be dynamic and so there's only one place where you determine the tested tokens